### PR TITLE
Action to modify services

### PR DESCRIPTION
**Pull request by bhimsenpadalkar**
### Checklist
- [x] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [x] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [x] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [ ] I've updated the documentation if necessary.

### Motivation and Context
This is against the issue raised [here](https://github.com/fastlane/fastlane/issues/10007)

### Description
This will add the action to modify the services.

### CODE REVIEW

File: fastlane/lib/fastlane/actions/modify_services.rb [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+                                       is_string: false,
+                                       default_value: {},
+                                       verify_block: proc do |value|
+                                         allowed_keys = Produce::DeveloperCenter::ALLOWED_SERVICES.keys
+                                         UI.user_error!("enable_services has to be of type Hash") unless value.kind_of?(Hash)
+                                         value.each do |key, v|
+                                           UI.user_error!("The key: '#{key}' is not supported in `enable_services' - following keys are available: [#{allowed_keys.join(',')}]") unless allowed_keys.include? key.to_sym
+                                         end
+                                       end),
+          FastlaneCore::ConfigItem.new(key: :team_id,
```
Comment (KrauseFx) [inline fastlane/lib/fastlane/actions/modify_services.rb]: We'd also need the team name here, like see `produce/options.rb` 👍

File: fastlane/lib/fastlane/actions/modify_services.rb [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+                                       optional: true,
+                                       default_value: CredentialsManager::AppfileConfig.try_fetch_value(:team_id),
+                                       verify_block: proc do |value|
+                                         ENV["FASTLANE_TEAM_ID"] = value.to_s
+                                       end)
+        ]
+      end
+
+      def self.author
+        "Bhimsen Padalkar"
```
Comment (KrauseFx) [inline fastlane/lib/fastlane/actions/modify_services.rb]: Please use your Twitter or GitHub username 👍

File: fastlane/lib/fastlane/actions/modify_services.rb [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+      end
+
+      def self.is_supported?(platform)
+        platform == :ios
+      end
+
+      def self.example_code
+        [
+          'modify_services(
+            username: "bhimsenp@thoughtworks.com",
```
Comment (KrauseFx) [inline fastlane/lib/fastlane/actions/modify_services.rb]: Do you work at thoughtworks? Do you want to use a dummy email address instead?

**Comment by KrauseFx** [comment]
This is great, that's a missing piece to use this produce feature from within fastlane 👍 I added a few comments before it's ready to be merged

**Comment by bhimsenpadalkar** [comment]
Incorporated the feedback

**Review by KrauseFx — APPROVED** [review]
Nice, this is great. One quick question before we merge, have you tried running this action locally? Does it work as expected?

**Comment by KrauseFx** [comment]
Awesome, thanks for your contribution 👍

**Comment by bhimsenpadalkar** [comment]
It doesn't require immediate release. It can go in next release.

**Comment by KrauseFx** [comment]
Already live 🚀 Thanks again

**Pull request merged**